### DISCUSSION TITLE
TimedSubdomainModifier

### DISCUSSION THREAD

**Discussion by PEI0214** [idaholab/moose#28521] [Q&A General]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

Hello
When I modified the subdomain using TimedSubdomainModifier, the displacement continued to increase at each time step. When I use CoupledVarThresholdElementSubdomainModifier modify domain, this will not happen. What causes this? Do you have any good suggestions?

**Comment by GiudGiud** [comment]
hello

instead of looking at the displacement, look at the subdomain assignments, to make sure the two objects are actually moving the elements the same way

Guillaume

**Reply by PEI0214** [reply]
The subdomain assignment is the same for both methods. The subdomain assignment is the same for both methods. Only the displacement is inconsistent. Curiously, after the subdomain modification is complete, test 1 displacement increases while other variables (such as stress) remain unchanged.

**Reply by PEI0214** [reply]
Hello, do you know what caused this situation?

**Reply by GiudGiud** [reply]
We have just updated the element subdomain modifier (last Thursday). So it is worth updating moose and seeing if the behavior is more consistent now

**Reply by GiudGiud** [reply]
any update on this?
btw I am looking a figure on the use of CoupledVarThresholdElementSubdomainModifier / MeshModifiers if you have a good one I would love to be able to feature it at a coming conference (ANS Winter 2024)
